### DISCUSSION TITLE
Week Planner: Can I change a week?

### DISCUSSION THREAD

**Discussion by queezz** [ivan-lednev/obsidian-day-planner#425] [Q&A]
The week planner is very helpful. It would be even better if I could change the week to review past plans and plan ahead, not just for the current week. Is that already possible and did I miss this somewhere in controls?

**Comment by ivan-lednev** [comment]
Yes you can. If you don't see the controls, make sure this option is enabled:
![image](https://github.com/ivan-lednev/obsidian-day-planner/assets/41428836/244fc3da-6e00-4359-8f0a-296bc6a2c260)
